test/distributed/org/apache/cassandra/distributed/util/NodetoolUtils.java

```diff
@@ -44,28 +44,17 @@ public static NodeToolResultWithOutput nodetool(IInvokableInstance inst, String.
     public static NodeToolResultWithOutput nodetool(IInvokableInstance inst, boolean withNotifications, String... args)
     {
         return inst.callOnInstance(() -> {
```

Comment:
not necessary to run it with `callOnInstance`. we can get the nodetool result by calling `nodetoolResult` on a node. For instance,
```java
NodeToolResult ringResult = CLUSTER.get(1).nodetoolResult("ring");
```

Reply:
Ok so just to make sure I follow, this whole method could be replaced with just

```
return inst.nodetoolResult(withNotifications, args);
```

?

In this case I think we can indeed get rid of `NodetoolUtils` and also `NodeToolResultWithOutput`. I will try it out and let you know how it goes.

Reply:
There is also this comment in NodetoolUtils which I don't think is used at other places so I also kind of prefer to remove it after double-checking my statement as I am multitasking now :D - 
"// Prefer to use the NodeToolResult that includes output since version 0.0.5
// CASSANDRA-16057"
I don't think NodetoolUtils is used anywhere else"

Reply:
Confirmed, it was very easy to get rid of these classes. Done in db2f975. Thank you for the suggestion!